python/servo/post_build_commands.py

```diff
@@ -81,7 +81,7 @@ class PostBuildCommands(CommandBase):
     def run(self, params, release=False, dev=False, android=None, debug=False, debugger=None,
             headless=False, software=False, bin=None, emulator=False, usb=False, nightly=None):
         env = self.build_env()
-        env["RUST_BACKTRACE"] = "1"
+        env["RUST_BACKTRACE"] = "full"
```

Comment:
Maybe we should do like `env["RUST_BACKTRACE"] = env["RUST_BACKTRACE"] or "1"` 🤔?

Reply:
There are several other places where we set RUST_BACKTRACE. We should probably update those ones as well.

Reply:
That being said, your idea probably makes sense so that we can override this behaviour on our CI. We probably don't want the colours in the logs.

Reply:
@CYBAI There are three other places we set RUST_BACKTRACE in post_build_commands.py and testing_commands.py; can we update those as well?

Reply:
@jdm Oops, I saw it here. Sorry for missing it! Will update, thanks!